Sources/StreamChat/WebSocketClient/EventMiddlewares/ChannelVisibilityEventMiddleware.swift

```diff
@@ -25,6 +25,7 @@ struct ChannelVisibilityEventMiddleware: EventMiddleware {
                 
                 if event.isHistoryCleared {
                     channelDTO.truncatedAt = event.createdAt.bridgeDate
+                    channelDTO.previewMessage = session.preview(for: event.cid)
```

Comment:
Good catch 👍

Reply:
Sorry, un-resolving because it's better to have everything related to channel preview in one place. We're doing it in `saveEvent -> updateChannelPreview`.

Reply:
done